groups.py

```diff
@@ -705,6 +706,18 @@ def group_user_exists(ctx, group_name, username, include_readonly):
         return False
 
 
+@rule.make(inputs=[0], outputs=[1])
+def rule_user_exists(ctx, username):
+    """Rule wrapper to check if a user exists.
```

Comment:
It'd be nice to also add "('rodsuser' or 'rodsadmin')" for better clarification in the doc strings